frontend/src/components/Alerts/InfoAlertContent.tsx

```diff
@@ -0,0 +1,47 @@
+import { Icon, Typography } from '@equinor/eds-core-react'
```

Comment:
currently just on massive "Add skip auto missions feature" commit. Can it be split? Eg this file doesn't have a clear connection with that feature